File: r/Calculator.R
print("R Calculator v1.0")
print("")
print("Options")
print("1. Addition")
print("2. Subtraction")
print("3. Multiplication")
print("4. Division")
option = readline(prompt="Enter option: ")

print("")
a = strtoi(readline(prompt="Enter 1st input: "))
b = strtoi(readline(prompt="Enter 2nd input: "))

print("")
if (option == "1") {
  r = a+b
  print(paste("Result:", a, "+", b, "=", r))
} else if (option == "2") {
  r = a-b
  print(paste("Result:", a, "-", b, "=", r))
} else if (option == "2") {
  r = a*b
  print(paste("Result:", a, "*", b, "=", r))
} else if (option == "2") {
  r = a/b
  print(paste("Result:", a, "/", b, "=", r))
} else {
  print("Error: invalid choice.")
}

File: octave/Calculator.m
disp('Calculator v1.0')
disp('Copyright (C) 2020 miniprime1')
disp(' ')
disp('Options')
disp('1. Add')
disp('2. Subtract')
disp('3. Multiply')
disp('4. Divide')
opt = input('Enter choice: ');
disp(' ')
if (opt == 1)
    x = input('Enter 1st input: ');
    y = input('Enter 2nd input: ');
    disp(' ')
    disp('Result')
    fprintf("%f + %f = %f", x, y, x+y);
elseif (opt == 2)
    disp('Input')
    x = input('Enter 1st input: ');
    y = input('Enter 2nd input: ');    
    disp(' ')
    disp('Result')
    fprintf("%f - %f = %f", x, y, x-y);
elseif (opt == 3)
    x = input('Enter 1st input: ');
    y = input('Enter 2nd input: ');
    disp(' ')
    disp('Result')
    fprintf("%f * %f = %f", x, y, x*y);
elseif (opt == 4)
    x = input('Enter 1st input: ');
    y = input('Enter 2nd input: ');
    disp(' ')
    disp('Result')
    fprintf("%f / %f = %f", x, y, x/y);
else
    disp('Error: Invalid choice')
end

clear
